Add PDB format option to /debug flag

TyOverby opened the pull request.

TyOverby commented:
CC @agocke @AlekseyTs @jaredpar @TyOverby @VSadov @tmat
Please review

jaredpar commented on src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs:
```diff
@@ -1139,6 +1136,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 OutputFileName = outputFileName,
                 PdbPath = pdbPath,
                 EmitPdb = emitPdb,
+                // TODO: pipe debugInformationFormat through this
```
What work is left here?

TyOverby replied:
Oops.  I added that back before I learned that the format was already passed through on line 1116.  It should be removed; there's no more work left.

jaredpar commented:
:+1:

TyOverby commented:
I removed the todo comment.  Can I manually merge this right now?  The only change is that I removed the comment, and 4/5 tests passed on the first commit (the only one failing was a flakey ETA test).
